### PR TITLE
Use react-science confirmation dialog

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
I couldn't migrate modals in `KeysListenerTracker` and `FilterTable` because the modal have 3 options, so we will able to use `react-science` after improving [`ConfirmModal`]( https://github.com/zakodium-oss/react-science/issues/525)

### CODE REVIEW

File: src/component/panels/IntegralsPanel/IntegralPanel.tsx [hunk 1]
```diff
@@ -195,6 +193,34 @@ function IntegralPanelInner({
...
+            display: 'flex',
+            flex: '1 1 0%',
+            alignItems: 'center',
+            justifyContent: 'center',
+            fontWeight: 'bold',
+            margin: 25,
+            fontSize: 14,
+            color: 'rgb(175, 0, 0)',
+          }}
+        >
```
Comment (targos) [inline src/component/panels/IntegralsPanel/IntegralPanel.tsx:220]: This is not ok. You're repeating the same styles over and over.
Either we accept the new style, or we improve the style of the component in react-science.

Comment (wadjih-bencheikh18) [reply src/component/panels/IntegralsPanel/IntegralPanel.tsx:220]: Okay I will try to remove it

Comment (wadjih-bencheikh18) [reply src/component/panels/IntegralsPanel/IntegralPanel.tsx:220]: With styling: 
image

Without:
image

I think we should update `ConfirmModal` style in r`eact-science`

Comment (wadjih-bencheikh18) [reply src/component/panels/IntegralsPanel/IntegralPanel.tsx:220]: the needed style
```tsx
style={
            fontWeight: 'bold',
            margin: 25,
            fontSize: 14,
            color: 'rgb(175, 0, 0)',
}
```

Comment (targos) [reply src/component/panels/IntegralsPanel/IntegralPanel.tsx:220]: No, make a PR to `react-science` (with stories if needed)

**Comment by lpatiny** [comment]
If we press Enter when the dialog is there we get a focus around the dialog and this should not be the case.

image

**Comment by targos** [comment]
This PR will need to be updated now that the styling issues are fixed in react-science.

**Comment by wadjih-bencheikh18** [comment]
> This PR will need to be updated now that the styling issues are fixed in react-science.

The outline problem is solved but there is an issue in the new release
https://github.com/zakodium-oss/react-science/issues/575

**Comment by wadjih-bencheikh18** [comment]
This PR need to be closed

**Pull request closed without merging**